tests/installer/installer_test.go

```diff
@@ -0,0 +1,68 @@
+package cos_test
+
+import (
+	"github.com/rancher-sandbox/cOS/tests/sut"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("cOS Installer tests", func() {
+	var s *sut.SUT
+
+	BeforeEach(func() {
+		s = sut.NewSUT()
+		s.EventuallyConnects()
+	})
+
+	AfterEach(func() {
+		if CurrentGinkgoTestDescription().Failed {
+			s.GatherAllLogs()
+		} else {
+			// Trash disk so we boot from iso after each test
+			s.EmptyDisk("/dev/sda")
+			// Reboot so it boots from iso
+			s.Reboot()
+		}
+	})
+	Context("Using legacy bios", func() {
+		Context("generic tests", func() {
+			It("booted from iso", func() {
```

Comment:
mmm and maybe we can do these assertion right after we empty the disk, before each suite?